acceptance-tests/lib/env.js

```diff
@@ -52,39 +50,47 @@ const envOverrides = getTruthyValuesOnly({
 });
 
 const getTestConfig = () => {
-  // Test-specific Overrides > Command-line Args > Env vars
-  const config = Object.assign(
-    {},
-    envOverrides,
-    localOverrides,
-    localTestOverrides
-  );
+  // Command-line Args > Env vars
+  const config = Object.assign({}, envOverrides, argsOverrides);
 
-  if (!config.portalId)
+  if (!config.portalId) {
     throw new Error(
-      'accountId must be defined.  Either set the ACCOUNT_ID environment variable, or use the --accountId flag to pass it in.'
+      'accountId must be defined. Either set the ACCOUNT_ID environment variable or use the --accountId flag to pass it in.'
     );
+  }
 
-  if (!config.cliPath)
-    throw new Error(
-      'cliPath must be defined.  Either set the CLI_PATH environment variable or use the --cliPath flag to pass it in.'
-    );
+  if (!config.cliPath) {
+    const defaultPath = path.join(process.cwd(), DEFAULT_CLI_PATH);
+
+    if (existsSync(defaultPath)) {
+      config.cliPath = defaultPath;
+    } else {
+      throw new Error(
+        'cliPath must be defined. Either set the CLI_PATH environment variable or use the --cliPath flag to pass it in.'
+      );
+    }
+  }
 
-  if (!config.personalAccessKey)
+  if (!config.personalAccessKey) {
     throw new Error(
-      'No valid auth for personalAccessKey was found. Set the PERSONAL_ACCESS_KEY environment variable or use the --personalAccessKey flag to pass it in.'
+      'personalAccessKey must be defined. Either set the PERSONAL_ACCESS_KEY environment variable or use the --personalAccessKey flag to pass it in.'
     );
+  }
 
-  if (!config.githubToken)
-    throw new Error(
-      'githubToken must be defined.  Either set the GITHUB_TOKEN environment variable, or use the --githubToken flag to pass it in.'
+  if (!config.githubToken) {
```

Comment:
I don't think this is a required field. I was able to successfully run the acceptance tests without this, so I'm making it a recommended field instead